### ISSUE TITLE
Clipboard for data points

### ISSUE THREAD

**Issue by mmmavis** [mozilla/mozfest-program#465]
**[ Google Spreadsheet Row Number ]** 441
**[ Facilitator ]** Anatoly Bondarenko
### Description

Most of data-driven web applications lacks methods to remember interesting data points and little insights which we encounter during data exploration. In this open session we'll use some examples (choosed or made by facilitator) in existing applications to clarify the problem and to brainstorm how to give users a convenient, unobtrusive ways to effectively remember their results during data analysis. Possible methods spread from simple "clipboard" or "shopping cart" to "all states remembered" metaphors.
### Agenda

Basically it could be a short demonstration of examples (30 min) and brainstorm session on an ideas in small groups (30 min), and formulation of results (30 min)
### Participants

After some size we should make a groups with a size of approx 5-7 people to brainstorm and discuss an ideas in each of the group.
### Outcome

My hope is that after this session participants will embrace an idea of 'clipboard memory' for data visualizations and start to implement the same methods in own work.

**Comment by thornet** [comment]
cc @thornet

**Comment by marcwalsh-zz** [comment]
cc @ryanpitts

**Comment by thornet** [comment]
cc @kaythaney @arlissc as well for open data / science

**Comment by Saallen** [comment]
We appreciated this submission but unfortunately this session does not fit within the narrative of our Space for 2015- Hope you will submit another session for 2016
